tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -41,7 +41,11 @@ include $(RIOTBASE)/Makefile.include
 
 .PHONY: dhcpv6_server
 
-dhcpv6_server: IFACE := ${IFACE}
+$(call target-export-variables,dhcpv6_server test-with-config/check-config,IFACE)
```

Comment:
```suggestion
# export IFACE variable to environment of dhcpv6_server and test-with-config/check-config rules
$(call target-export-variables,dhcpv6_server test-with-config/check-config,IFACE)
```
I think this improves the understanding of this call

Reply:
Done. I also added a link to the respective [doc](https://doc.riot-os.org/build-system-basics.html#variable-declaration-guidelines) for more context :-).